src/adhocracy_mercator/adhocracy_mercator/workflows/test_init.py

```diff
@@ -0,0 +1,168 @@
+    assert workflow.state_of(context) is None
+    workflow.initialize(context)
+    assert workflow.state_of(context) is 'draft'
+    workflow.transition_to_state(context, request, 'announce')
+    assert workflow.state_of(context) is 'announce'
+    workflow.transition_to_state(context, request, 'participate')
+    assert workflow.state_of(context) is 'participate'
+    workflow.transition_to_state(context, request, 'frozen')
+    assert workflow.state_of(context) is 'frozen'
+    workflow.transition_to_state(context, request, 'result')
+    assert workflow.state_of(context) is 'result'
+
+
+@fixture(scope='class')
+def app_anonymous(app_anonymous):
+    app_anonymous.base_path = '/mercator'
+    return app_anonymous
+
+@fixture(scope='class')
+def app_participant(app_participant):
+    app_participant.base_path = '/mercator'
+    return app_participant
+
+
+@fixture(scope='class')
+def app_god(app_god):
+    app_god.base_path = '/mercator'
+    return app_god
+
+@fixture(scope='class')
+def app_initiator(app_initiator):
+    app_initiator.base_path = '/mercator'
+    return app_initiator
+
+def _post_proposal_item(app_user, path='/',  name='') -> TestResponse:
+    from adhocracy_mercator.resources.mercator import IMercatorProposal
+    from adhocracy_core.sheets.name import IName
+    iresource = IMercatorProposal
+    sheets_cstruct = {IName.__identifier__: {'name': name}}
+    resp = app_user.post_resource(path, iresource, sheets_cstruct)
+    return resp
+
+def _post_proposal_version(app_user, path='/') -> TestResponse:
+    from adhocracy_mercator.resources.mercator import IMercatorProposalVersion
+    iresource = IMercatorProposalVersion
+    resp = app_user.post_resource(path, iresource, {})
+    return resp
+
+def _batch_post_full_sample_proposal(app_user) -> TestResponse:
+    subrequests = _create_proposal()
+    resp = app_user.batch(subrequests)
+    return resp
+
+def _do_transition_to(app_user, path, state) -> TestResponse:
+    from adhocracy_mercator.sheets.mercator import IWorkflowAssignment
+    data = {'data': {IWorkflowAssignment.__identifier__:\
+                         {'workflow_state': state}}}
+    resp = app_user.put(path, data)
+    return resp
+
+@mark.functional
+class TestMercatorProposalPermissionsAnonymous:
+
+    def test_cannot_create_proposal_item(self, app_anonymous):
+        resp = _post_proposal_item(app_anonymous, path='/', name='proposal1')
+        assert resp.status_code == 403
+
+    def test_cannot_create_proposal_per_batch(self, app_anonymous):
+        resp = _batch_post_full_sample_proposal(app_anonymous)
+        assert resp.status_code == 403
+
+    def test_cannot_create_proposal_per_batch_broken_token(
+            self, app_broken_token):
+        resp = _batch_post_full_sample_proposal(app_broken_token)
+        assert resp.status_code == 400
+
+
+@mark.functional
+class TestMercatorWorkflow:
+
+    ## the state is currently set to 'participate' when creating the mercator process
+
```

Comment:
instead of commenting add  mark.xfail with and explanation.
We should also extend the workflow api to reset the the state to the the inital state